inc/dashboard/grid.class.php

```diff
@@ -1194,6 +1194,40 @@ public function getAllDasboardCards($force = false): array {
             }
          }
 
+         foreach (get_declared_classes() as $itemtype) {
+            if (is_subclass_of( $itemtype, 'Item_Devices')) {
+               $clean_itemtype = str_replace('\\', '_', $itemtype);
+               $cards["bn_count_$clean_itemtype"] = [
+                  'widgettype' => ["bigNumber"],
+                  'group'      => _n('Device', 'Devices', 1),
+                  'itemtype'   => "\\$itemtype",
+                  'label'      => sprintf(__("Number of %s"), $itemtype::getTypeName()),
+                  'provider'   => "Glpi\\Dashboard\\Provider::bigNumber$itemtype",
+                  'filters'    => array_merge([
+                     'dates',
+                     'dates_mod',
+                  ], $add_filters_fct($itemtype::getTable()))
+               ];
+            }
+         }
+
+         foreach (get_declared_classes() as $itemtype) {
```

Comment:
Same remark here, you could rely on `$CFG_GLPI['device_types']`.

Reply:
I will adapt the code, thx for the comments